daru.gemspec

```diff
@@ -51,6 +51,7 @@ EOF
 
   spec.add_runtime_dependency 'reportbuilder', '~> 1.4'
   spec.add_runtime_dependency 'spreadsheet', '~> 1.0.3'
+  spec.add_runtime_dependency 'dbi'
```

Comment:
Moved this to a development dependency. I also removed the require 'dbi' from within the method. Because the user has to pass in a DBI::DatabaseHandle, it should be installed before calling from_sql.